src/PureCloud.Client/Apis/WorkforceManagementApi.cs

```diff
@@ -0,0 +1,95 @@
+using System.Net.Http.Json;
+using Microsoft.Extensions.Options;
+using PureCloud.Client.Contracts;
+using PureCloud.Client.Json;
+using PureCloud.Client.Models;
+
+namespace PureCloud.Client.Apis;
+
+public class WorkforceManagementApi : IWorkforceManagementApi
+{
+    private readonly IHttpClientFactory _httpClientFactory;
+    private readonly PureCloudJsonSerializerOptions _options;
+
+    public WorkforceManagementApi(
+        IHttpClientFactory httpClientFactory,
+        IOptions<PureCloudJsonSerializerOptions> options)
+    {
+        _httpClientFactory = httpClientFactory;
+        _options = options.Value;
+    }
+
+    // get methods
+
+    public async Task<List<UserScheduleAdherence>> GetAdherenceAsync(List<string> userId, CancellationToken cancellationToken = default)
+    {
+        var client = _httpClientFactory.CreateClient(PureCloudConstants.PureCloudClientName);
+
+        // Build query parameters for userId list
+        var queryParams = new List<string>();
```

Comment:
Can we use `NameValueCollection` instead of a list and then use `UriHelper.GetUri("", parameters)`